method/SpaceFlow/method_spaceflow.py

```diff
@@ -155,12 +166,15 @@ def get_anndata(args):
     nn = 15
 
 # Raise a warning that clustering is based on resolution and not n_clusters
-warnings.warn("The `n_clusters` parameter was not used; config['res'] used instead.")
+# warnings.warn("The `n_clusters` parameter was not used; config['res'] used instead.")
 
 # Segment the domains given the resolution
-sf.segmentation(domain_label_save_filepath=label_file, n_neighbors=nn, resolution=res)
```

Comment:
Does this function use Leiden clustering?

Reply:
Yes. Check here:

https://github.com/hongleir/SpaceFlow/blob/58d1ab03f1538791f710db840d36a08f396491fb/SpaceFlow/SpaceFlow.py#L289-L314